### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by shanel262**
Inserted section to tell user to insert their github token. If authentication fails then fuge@=>0.6.1 should be used. #26

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,14 +77,19 @@ In the `./nodezoo-system` folder run,
 fuge build fuge/system.yml
 ```
 
-### 6. Start the infrastructure
+### 6. Insert GitHub token
+For the GitHub service to work you must insert your GitHub personal access token in to the sample.env file located in fuge/env.
```
Comment (mcdonnelldean) [inline README.md]: You need to copy sample.env to system.env

File: README.md [hunk 2]
```diff
@@ -77,14 +77,19 @@ In the `./nodezoo-system` folder run,
...
+
+__Note:__ If GitHub fails to authenticate you then it may be because Fuge at minimum version 0.6.1 is required.
```
Comment (mcdonnelldean) [inline README.md]: Change to, Fuge 0.6.1 or later is required, put it at the top of the file when you install fuge.

File: README.md (hunk 1 above)
Comment (shanel262) [inline README.md]: Would we not just use system.env instead of needing to copy everything over? @mcdonnelldean

Comment (mcdonnelldean) [inline README.md]: system.env is not checked in, so you can put all your private stuff in it and not accidentally commit.

File: README.md (hunk 2 above)
Comment (mcdonnelldean) [inline README.md]: Line 24, put it there

File: README.md (hunk 1 above)
Comment (shanel262) [inline README.md]: Ah right I see, I'll make the change :)

**Pull request merged**